### DISCUSSION TITLE
Number of tubes in Plug Flow Reactor

### DISCUSSION THREAD

**Discussion by SaifTM** [DanWBR/dwsim#445] [Q&A] [answered]
Hi!
I am working on simulating Cumene process in [https://pubs.acs.org/doi/abs/10.1021/ie9011535](url)
I have an issue when specifying the number of tubes . I get error message "High Pressure Drop".

I have checked the source code and I believe the error is due to the molar flows not divided by the number of tubes. 
Please, confirm.

Thanks!

**Comment by DanWBR** [comment, marked as answer]
unbelievable. thanks for the info, @SaifTM. fixed in https://github.com/DanWBR/dwsim/commit/73f62e0ed9b74d8191f6d0e2780fe84a53cbefb6
